provisioners.proto

```diff
@@ -166,13 +167,21 @@ message K8sSAProvisioner {
 
 message SSHPOPProvisioner {}
 
+message SCEPDecrypter {
+	config.KMS kms = 1;
+	bytes decrypterCertificate = 2;
+	string decrypterKey = 3;
+	string decrypterKeyPassword = 4;
```

Comment:
Bytes per Configuration.password.

Reply:
Do we need a `decrypterKeyPasswordFile` too?

Reply:
For a file it should be a flag `--decrypter-password-file`

Reply:
For now I'll only support a password stored as bytes. If a password file is used (locally), the contents have to be read first and then stored in the provisioner.